flow/FastAlloc.cpp

```diff
@@ -248,18 +248,32 @@ long long FastAllocator<Size>::getActiveThreads() {
 #if FAST_ALLOCATOR_DEBUG
 static int64_t getSizeCode(int i) {
```

Comment:
I guessed at what made sense here but I'm not sure I understand what this function does. Everything is a power of 2 except 96 and the return value isn't the exponent or something.

Reply:
For closure: @sfc-gh-tclinkenbeard pointed me at this change: https://github.com/apple/foundationdb/commit/007abbc45b4dbf2208ea7f47b99b0c567c94b2ce